dist/search-insights.min.js

```diff
@@ -1 +1 @@
Object;t&&((window[t].queue||[]).forEach(function(t){var n=t[0],r=t[1];n&&"function"==typeof e[n]&&e[n](r)}),window[t]=function(t,n){e[t](n)})}.bind(this),this.sendEvent=e.bind(this),this.init=function(e){if(!e)throw new Error("Init function should be called with an object argument containing your apiKey and applicationID");if(!e.apiKey||"string"!=typeof e.apiKey)throw new Error("apiKey is missing, please provide it so we can authenticate the application");if(!e.applicationID||"string"!=typeof e.applicationID)throw new Error("applicationID is missing, please provide it, so we can properly attribute data to your application");this._apiKey=e.apiKey,this._applicationID=e.applicationID,this._hasCredentials=!0}.bind(this),this.initSearch=function(e){if(!this._hasCredentials)throw new Error("Before calling any methods on the analytics, you first need to call the 'init' function with applicationID and apiKey parameters");if(!e)throw new Error("initSearch function requires an argument with getQueryID and hitsContainer arguments");if(!e.getQueryID||"function"!=typeof e.getQueryID)throw new Error("getQueryID must be a function that returns the queryID of the last search operation");this.getQueryID=e.getQueryID}.bind(this),this.click=function(e){if(!this._hasCredentials)throw new Error("Before calling any methods on the analytics, you first need to call the 'init' function with applicationID and apiKey parameters");if(!e)throw new Error("No params were sent to click function, please provide an objectID and position to be reported");if(!e.objectID)throw new Error("required objectID parameter was not sent, click event can not be properly attributed");if(!e.position)throw new Error("required position parameter was not sent, click event position can not be properly sent without");var t=e.queryID;if("function"!=typeof this.getQueryID||t||(t=this.getQueryID()||this._lastQueryID),!t)throw new Error("No queryID was retrieved, please check the implementation and provide either a getQueryID function\n    or call the search method that will return the queryID parameter");this.storageManager.storeClick(e.objectID,t);var n=Object.assign({},e,{queryID:t});this.sendEvent("click",n)}.bind(this),this.conversion=function(e){if(!this._hasCredentials)throw new Error("Before calling any methods on the analytics, you first need to call the 'init' function with applicationID and apiKey parameters");if(!e)throw new Error("No parameters were sent to conversion event, please provide an objectID");if(!e.objectID)throw new Error("No objectID was sent to conversion event, please provide an objectID");var t=e.queryID?e.queryID:this.storageManager.getConversionObjectID(e.objectID),n=Object.assign(e,{queryID:t});if(!t)throw new Error("No queryID was retrieved, please check the implementation and provide either a getQueryID function\n    or call the conversion method that will return the queryID parameter");this.sendEvent("conversion",n)}.bind(this),this._userID=c(),this.processQueue()}});
```

Comment:
I wonder why the dist is tracked on GitHub 🤔

Reply:
no sure why either. Any insights on this @JonasBa ?

Reply:
Yeah, there's no need. We can remove it.

Reply:
will do it in my next PR, stay tuned :)

Reply:
done here: https://github.com/algolia/search-insights.js/pull/21